### PR TITLE
Python 3.9 support

### PR DESCRIPTION
**Pull request by mbolivar-nordic**
Built and tested using zephyr menuconfig against Python 3.9rc1 on Windows 10 using Microsoft (R) C/C++ Optimizing Compiler Version 19.27.29111 for x64.

I'm not sure how we should release and version this. The README says we need a version bump, and we're currently at 2.1.0 on PyPI:

https://pypi.org/project/windows-curses/#history

I think this means we should release 2.2.0 and we should rebuild wheels for other Python versions? I'm conflicted about doing that because I don't know if I can replicate the build environment from previous versions properly and I don't want to fix what isn't broken.

If we decide to go that route I would want to drop Python 2 support at the very least.

Thoughts?

### CODE REVIEW

**Comment by mbolivar-nordic** [comment]
> I think this means we should release 2.2.0 and we should rebuild wheels for other Python versions?

@carlescufi this is what I'm going to assume unless you think otherwise. Please let me know.

**Comment by carlescufi** [comment]
> > I think this means we should release 2.2.0 and we should rebuild wheels for other Python versions?
> 
> @carlescufi this is what I'm going to assume unless you think otherwise. Please let me know.

Absolutely @mbolivar-nordic. It's time to release 2.2.0. About rebuilding the wheels, that I am less sure about. If they work is there an actual reason to rebuild them?

**Comment by carlescufi** [comment]
> If we decide to go that route I would want to drop Python 2 support at the very least.

Absolutely. Regardless of what we do, Python 2 support should be dropped.

**Comment by carlescufi** [comment]
@mbolivar-nordic is this ready to merge?

**Comment by mbolivar-nordic** [comment]
> @mbolivar-nordic is this ready to merge?

I haven't gotten any feedback one way or the other since asking for testers, so I lean towards just getting this out the door.

I've pushed two extra commits marking the release and updating the readme. Ready to merge as far as I am concerned.

**Comment by mbolivar-nordic** [comment]
> Ready to merge as far as I am concerned.

@carlescufi ping

File: README.md [hunk 1]
```diff
@@ -58,13 +58,11 @@ Build instructions
     builds wheel for by following the instructions at
     https://wiki.python.org/moin/WindowsCompilers.
 
-    Visual Studio 2019 will work for Python 3.5-3.8. For Python 3.5 support,
-    you will need to check `VC++ 2015.3 v140 toolset for desktop (x86,x64)`
-    during installation.
+    Visual Studio 2019 will work for Python 3.6-3.8.
```
Comment (carlescufi) [inline README.md:61]: So what about 3.9? We should document the toolchain you (will) use to compile those.

Comment (mbolivar-nordic) [reply README.md]: > So what about 3.9? We should document the toolchain you (will) use to compile those.

I missed that one, thanks. I used the same toolchain for everything so I amended this to say 3.9.

**Review by carlescufi — APPROVED** [review]

**Pull request merged**